spyder/plugins/maininterpreter/confpage.py

```diff
@@ -48,30 +48,17 @@ def __init__(self, plugin, parent):
                 valid_custom_list.append(path)
         self.set_option('custom_interpreters_list', valid_custom_list)
 
-        # Python executable selection (initializing default values as well)
+        # add custom_interpreter to executable selection
         executable = self.get_option('executable', get_python_executable())

```

Comment:
If I see correctly, removing this line would remove `executable` variable from scope.
It is used to get the current active interpreter and add it to the dropdown menu in `_add_to_custom_interpreters` below?

Reply:
You're right, my mistake. Then please change this to be

```python
executable = self.get_option('executable')
```

That's because the second parameter of `get_option` is a default value. But that's not necessary here since that value is set in `get_main_interpreter`